src/google-calendar.ts

```diff
@@ -42,26 +42,38 @@ function addButtonToQuickAdd(quickAddDialog: HTMLElement) {
     const tabPanel = document.createElement("content");
     tabPanel.setAttribute("role", "tabpanel");
     tabPanel.setAttribute("id", "jitsi_button_quick_add_content");
+
     tabPanel.innerHTML = `
-      <div class="fy8IH poWrGb">
+      <div class="fy8IH poWrGb"> 
```

Comment:
Sorry if i'm exhibiting ADD here... is there a significance to the extra space?

Reply:
ha! no significance at all!

I'm so used to prettier cleaning up my end of line spaces, but this one is inside a string so got missed :)